docs/k8s-helm.md

```diff
@@ -221,65 +302,90 @@ Throughput: 125
 #### Step 3 - Create a new PV
 
 Firstly, use the PV name you found in step 1 and the volume ID of volume you created in step 2 to run the following command to generate the YAML file for the new PV you are going to create:
+
 ```bash
 kubectl get pv pvc-17510f7b-8e66-472d-a1dc-4245b2f51e1a -oyaml | yq 'del(.metadata.creationTimestamp) | del(.metadata.resourceVersion) | del(.metadata.uid) | del(.status) | .spec.csi.volumeHandle = "vol-0d628e0096371cb67" | .metadata.name = "pvc-manual-vol-0d628e0096371cb67"' > new-pv.yaml
 ```
 
 Then use the YAML file to create the PV
+
 ```bash
 kubectl apply -f new-pv.yaml
 ```
 
 #### Step 4 - Make the proton pod use the new PV
 
 First, delete the existing PV
+
 ```bash
 kubectl delete pv pvc-17510f7b-8e66-472d-a1dc-4245b2f51e1a
 ```
+
 Even though it will show the PV is deleted, but the command will be blocked because the PV is currently in-use. So leave this along, and open a new terminal to run the next command.
 
 Next, delete the existing pvc
+
 ```bash
 kubectl delete pvc -n $NS proton-data
 ```
+
 Similarly, this command will also be blocked because the PVC is in-use. Leave it alone, and open a new terminal to run the next command.
 
 Finally, delete the proton pod
+
 ```bash
 kubectl delete pod -n $NS proton-0
 ```
+
 Once the pod is deleted, the previous commands will also be unblocked thus the existing PVC and PV will be deleted.
 
 Once the new pod is up and running, it will use the PV you previously created manually. You can double check by
+
 ```bash
 kubectl get pvc -n $NS proton-data -o=jsonpath='{.spec.volumeName}'
 ```
-It should return the name you used in your new PV, in this example it is `pvc-manual-vol-0d628e0096371cb67`.
 
+It should return the name you used in your new PV, in this example it is `pvc-manual-vol-0d628e0096371cb67`.
 
 ### Troubleshooting
 
 If something goes wrong, you can run the following commands to get more information.
+
 1. `kubectl get pods -n $NS`: Make sure all pods are in `Running` status and the `READY` is `1/1`.
 2. `kubectl logs <pod> -n $NS`: Try to check the logs of each pod to make sure there is no obvious errors.
 3. Run `kubectl cluster-info dump -n $NS` to dump all the information and send it to us.
 
 ## Configuration Guide
 
-You can customize the deployment by applying a YAML file with your preferred values. Each Helm chart version may have slightly different available configurations. Here are some common settings. For the full list of configurable values, please check the `README.md` and `values.yaml` in the [Helm chart package](https://github.com/timeplus-io/install.timeplus.com/tree/main/charts).
+You may want to customize the configurations of Timeplus Appserver or Timeplusd. Here is a quick example of how to modify the `values.yaml`. For the list of available configuration items, please refer to the docs of [Timeplus Appserver](https://docs.timeplus.com/server_config#appserver) and [Timeplusd](https://docs.timeplus.com/server_config#timeplusd).
```

Comment:
For inter-site link, pls use /server_config..., without the hostname. This will help the doc tool to detect deadlink and handle i18n, but this will introduce a difference between markdown in docs and in helm chart.

Reply:
updated link